config/locales/en.yml

```
@@ -27,6 +27,9 @@ en:
       next: "Next"
       previous: "Previous"
       nav_pagination_label: "Entry pagination"
+  rss_feed:
```

```suggestion
  rss_feeds:
```

This would be more consistent with the other keys